_pages/speaking-resources.md

```diff
@@ -7,7 +7,7 @@ title: Speaker Resources
 ---
 
 ## Information for Speakers
-* The Speaker Green Room will be in <strong>Santa Fe 4</strong> on Sunday-Wednesday.
+{% comment %}* The Speaker Green Room will be in <strong>Santa Fe 4</strong> on Sunday-Wednesday.{% endcomment %}
 * Please be in your presentation room at least 15 minutes before your talk to meet your session chair.
 * The Quiet Room will be in the <strong>Santa Fe 4</strong> on all conference days.
```

Comment:
comment out Santa Fe 4 in line 12 as well?

Reply:
Good catch, thanks!

I'm going to change it to something like "A Quiet Room will be available on all conference days"

Reply:
Done